source/aggregation-pipeline-builder.txt

```diff
@@ -31,6 +31,13 @@ aggregation pipeline. The :guilabel:`Preview of Documents in the
 Collection` section shows 20 documents randomly sampled from the chosen
 collection.
 
+.. note:: 
+
+   When you connect Compass to MongoDB instances hosted on 
+   `Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`_,
+   additional Atlas only stages ``$search`` and ``$searchMeta`` are 
```

Comment:
[suggestion]

Can we provide links to the $search and $searchMeta stages:

- [$search](https://www.mongodb.com/docs/atlas/atlas-search/query-syntax/#-search)
- [$searchMeta](https://www.mongodb.com/docs/atlas/atlas-search/query-syntax/#-searchmeta)

We can use the `:atlas:` role to simplify the syntax for these links.

Reply:
Sure thing, I added these as links and used the :atlas: role.